sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_recognize_content.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_content.py
+
+DESCRIPTION:
+    This sample demonstrates how to extact text and layout information from a document
```

Comment:
Are we going ahead with using the word `layout` instead of `content` in our samples?

Reply:
I'll switch all instances of layout to content